assets/aml-benchmark/components/src/dataset_downloader/main.py

```diff
@@ -47,7 +48,10 @@ def parse_args() -> argparse.Namespace:
         help="Split of the dataset to download; specify 'all' to download all splits.",
     )
     parser.add_argument(
-        "--url", type=str, default=None, help="URL of the dataset to download."
```

Comment:
Are we removing the functionality to download from a URL? Can you check if none of the P1 datasets were validated using it?

Reply:
We are good with this.